### ISSUE TITLE
Broken query box / inline query style

### ISSUE THREAD

**Issue by bernardmo** [phpmyadmin/phpmyadmin#13188]
Hi guys,

I've just upgraded PMA on my Centos server. After update I'm unable to use query box nor inline query editor (it seems that query is shifted out of it). Confirm box style is also broken. There are no errors in JS console. Tried to downgrade to 4.6.4 but it didn't help. 

Here are some pics:

![p2](https://cloud.githubusercontent.com/assets/3929434/25009373/c384e2f6-2066-11e7-8f15-d709dc244b5b.png)
![qb](https://cloud.githubusercontent.com/assets/3929434/25009375/c79f408e-2066-11e7-807e-fe21a0503e5f.png)
![cb](https://cloud.githubusercontent.com/assets/3929434/25009499/21a2a83c-2067-11e7-9f98-c39e892486ca.png)
![v](https://cloud.githubusercontent.com/assets/3929434/25009380/c9a0fb34-2066-11e7-8ad7-12a8d8190aea.png)

### Steps to reproduce
1. Open query box or inline editor

### Expected behaviour
Should be able to write query

### Actual behaviour
Can't write query

### Server configuration
**Operating system**:
Centos 6

**Web server:**
nginx 

**Database:**
MariaDB

**PHP version:**
7.0.7

**phpMyAdmin version:**
 4.7.0

### Client configuration
**Browser:**
Chrome/Firefox

**Operating system:**
Ubuntu

**Comment by udan11** [comment]
Have you tried clearing the cache? Maybe you can use private or incognito mode?

**Comment by bernardmo** [comment]
I did (hard reload, cache clear, incognito mode, other browser) and issue is still there

**Comment by devenbansod** [comment]
Hi @bernardmo , are you able to reproduce this on the latest bug-fix version or our [demo server](https://demo.phpmyadmin.net/)?

**Comment by bernardmo** [comment]
Demo server works fine. My server doesn't and I have no idea what did I mess up there. Tried hard reload and disable cache in chrome. In the meantime I've migrated PMA to other server and it works find there (but used v4.4), 

![version](https://user-images.githubusercontent.com/3929434/29796674-c316bf6c-8c52-11e7-98ad-693fa5335a7b.png)
![nocache](https://user-images.githubusercontent.com/3929434/29796673-c316a1d0-8c52-11e7-9cc8-60c43b7b2c37.png)

Since this is not reproducible anywhere except on that specific server maybe we can close the task.
